### PR TITLE
fix(mizrahi-scraper): update selectors to make it work

### PR DESCRIPTION
**Pull request by naorpeled**
credits to @Asafb26 and @dumsky

closes #669

### CODE REVIEW

**Comment by naorpeled** [comment]
@esakal I combined all the fixes and also added a small fix for a scenario I had in the UI

**Comment by esakal** [comment]
@naorpeled awesome thanks!
@naorpeled  @asfktz @dumsky As I don't have mizrahi account I'm waiting for your approval to merge it.
Let me know when it is ready and approved by you

File: src/scrapers/mizrahi.ts [hunk 1]
```diff
@@ -57,8 +57,8 @@ const afterLoginSelector = '#dropdownBasic';
 const loginSpinnerSelector = 'div.ngx-overlay.loading-foreground';
 const accountDropDownItemSelector = '#AccountPicker .item';
 const pendingTrxIdentifierId = '#ctl00_ContentPlaceHolder2_panel1';
-const emailUpdatePageHebrewTitle = "עדכון כתובת דוא''ל";
-const emailUpdatePageEnglishTitle = 'Update email address';
```
Comment (naorpeled) [inline src/scrapers/mizrahi.ts:60]: The email update page is now an embedded iframe and we no longer have an indication of it other than the url which is not really future proof.
Therefore, I added the selectors of something that will be more likely more useful (the checking account tab).

**Comment by naorpeled** [comment]
@esakal 
Hey,
how are you doing?

I tested the changes locally again,
adjusted the code a bit (left a review comment above that describes it).

I think we should push the changes for now because the master's version of the scraper is not working anyways,
what do you think?

**Comment by esakal** [comment]
@naorpeled I agree, it is better to have something working in master.
To let @asfktz @dumsky provide feedback i'll wait till tomorrow evening, if no one will raise any rejections/suggestions we will proceed and deploy this PR.

**Comment by naorpeled** [comment]
> @naorpeled I agree, it is better to have something working in master. To let @asfktz @dumsky provide feedback i'll wait till tomorrow evening, if no one will raise any rejections/suggestions we will proceed and deploy this PR.

Sounds awesome, thanks!

**Comment by asfktz** [comment]
Hmm.. Not sure what I've got credit for
Maybe you meant mention something else? 🤔

**Comment by naorpeled** [comment]
> Hmm.. not sure what I've got credit for. Maybe you meant to mention @Asafb26 ?

Yup, my bad 🤦

**Comment by esakal** [comment]
@naorpeled I'm going to merge it in the coming hour. Ideally I prefer that someone will do a second verification but this issue is open for too long. If there will be any issue with this PR we will know soon :)

**Pull request merged**